rpki-validator-app/src/main/scala/net/ripe/rpki/validator/views/TrustAnchorMonitorView.scala

```diff
@@ -92,6 +91,38 @@ class TrustAnchorMonitorView(ta: TrustAnchor, trustAnchorValidations: TrustAncho
     badge(level, number.toString, number == 0)
   }
 
+
+  def renderFetchDetails = {
+    <h3>Fetch error</h3>
+      <table id="fetch-details-table" class="zebra-striped" style="display: none;" data-source={ s"${Tabs.TrustAnchorMonitorTab.url}/fetch-detail/${ta.identifierHash}" }>
+        <thead>
+          <tr>
+            <th>URI</th>
+            <th>Message</th>
+          </tr>
+        </thead>
+        <tbody>
+        </tbody>
+      </table>
+      <script>
+        {
```

Comment:
i think the commented code could be removed. It will still be in the git history afterwards.